index.html

```diff
@@ -0,0 +1,622 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>TNW</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/ea61045147.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Source+Sans+Pro:wght@200&display=swap" rel="stylesheet"> 
+  <link href="https://fonts.googleapis.com/css2?family=Oswald&display=swap" rel="stylesheet"> 
+</head>
+<body>
+  <!-- Navigation bar -->
+  <header>
+    <div class="nav-top">
+      <nav>
+        <ul>
+          <li><a href="#">News</a></li>
+          <li><a href="#">
+            Events ▾
+            <div class="hidden">
+              <ul>
+                <li>TNW2020</li>
+                <li>Couch Conference</li>
+              </ul> 
+            </div>   
+          </a></li>
+          <li><a href="#">
+            Business ▾
+            <div class="hidden">
+              <ul>
+                <li>Index</li>
+                <li>Programs</li>
+              </ul>
+            </div>
+          </a></li>
+          <li><a href="#">AMAs</a></li>
+          <li><a href="#">Spaces</a></li>
+          <li><a href="#">Terms & Conditions</a></li>
+        </ul>
+        <ul>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Advertise</a></li>
+          <li><a href="#">Jobs</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+    </div>
+    <div class="nav-bottom">
+      <div class="nav-bottom-1">
+        <div class="logo-nav"></div>
+        <div class="nav-bottom-text">
+          <ul>
+            <li><a href="#">Latest</a></li>
+            <li><a href="#">Hard fork</a></li>
+            <li><a href="#">Plugged</a></li>
+            <li><a href="#">Readme</a></li>
+            <li><a href="#">Growth Quarters</a></li>
+            <li><a href="#">Shift</a></li>
+            <li><a href="#">Neural</a></li>
+          </ul>
+        </div>
+      </div>
+      <div class="nav-bottom-2">
+        <div class="nav-bottom-2-icon">
+          <i class="fas fa-search"></i>
+        </div>
+        <div class="nav-bottom-2-icon nav-icon">
+          <i class="fas fa-bars"></i>
+        </div>
+      </div>
+    </div>
+  </header>
+  <section>
+    <div class="main">
+
+      <!-- Heading -->
```

Comment:
Use the main tag to wrap elements within the main section [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp)